reference/entry.json

```diff
@@ -50,10 +50,18 @@
         "description": "Returns the details of the event for the given `eventId`."
       }
     },
-    "/events/{eventId}/bookingAvailability": {
+    "/events/{eventId}/booking-availability": {
       "parameters": [
         {
-          "$ref": "#/components/parameters/eventId"
+          "name": "eventId",
```

Comment:
Why was this also changed?

Reply:
That was not intended, thanks for spotting it. Probably Stoplight Studio that removed the reference for some reason in the editor. GUIs 🤷‍♂️

Reply:
I've reverted it and also fixed other places where there was an inline parameter for `eventId` or `placeId` instead of a reference to the global parameter definition.